### ISSUE TITLE
Would you share the detection result json file on ActivityNet?

### ISSUE THREAD

**Issue by JacobKong** [yjxiong/action-detection#19]
Dear Author:

I am reading your papers:
1. A Pursuit of Temporal Accuracy in General Activity Detection
2. Temporal Action Detection with Structured Segment Networks

Thanks for your great works!

Currently, I am evaluating my detection result on one out of five subsets of ActivityNet 1.2 validation set. However, on one has ever release the results on these five subsets respectively.

I am wondering would you share your detection result json file of these two works on ActivityNet 1.2 or 1.3 validation set with me?

If yes, can you email the file to weijie.kong@pku.edu.cn or just paste down below? That means a lot!

Thanks a lot!

**Comment by yjxiong** [comment]
You can generate the detection results by following the instructions and using the provided pre-trained models.
